src/Pages/Books.js

```diff
@@ -17,7 +17,7 @@ export default function Books() {
   ];
   return (
     <main>
-      <ul className="booksList">
+      <ul className="BooksList">
         {
           testData.map((book) => (
             <Book
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=Make%20sure%20that%20list,to%20consume%20the%20state.), Please Make sure that list of books displays books from Redux store using `useSelector` function.

Reply:
Done. Thanks a lot!